### PR TITLE
Support for dummy JWT with 000 tenantID

### PR DESCRIPTION
**Pull request by hyfather**
Rudimentary dummy JWT support.
Sample usage here, seems to be working: https://github.com/Polarishq/log-query/commit/7f70cbc1d180c08d0ce42a811f27a8d9f390cd2b

Please review & merge: @declanshanaghy, @bk10287, @melting, @ledio

### CODE REVIEW

**Comment by hyfather** [comment]
Example usage: https://github.com/Polarishq/log-query/commit/7f70cbc1d180c08d0ce42a811f27a8d9f390cd2b

**Comment by hyfather** [comment]
@declanshanaghy PR feedback addressed

**Comment by bk10287** [comment]
What do you think about using something like this: https://github.com/dgrijalva/jwt-go? It seems like we are going down the path of handling the JWT parsing/validating ourselves, no?

**Comment by hyfather** [comment]
@bk10287 -- yep, that is what we will be using eventually. Currently, we are not parsing jwt at all, just setting a zeroed out jwt.

File: framework/jwt.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package framework
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type jwtContextKey string
+
+// JWTKey is used to stroe and retrieve the token from the context
```
Comment (declanshanaghy) [inline framework/jwt.go]: Typo here in "stroe"

File: framework/jwt.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+var JWTKey jwtContextKey
+
+func init() {
+	JWTKey = "polarisjwt"
+}
+
+// PolarisJWT is used by all Polaris services to infer auth information
+type PolarisJWT struct {
```
Comment (declanshanaghy) [inline framework/jwt.go]: Only add fields we're actually using

**Comment by declanshanaghy** [comment]
Looks good. Just a few small tweaks needed.

Also, make sure to update the Readme.md with example usage.

File: framework/jwt.go (hunk 2 above)
Comment (melting) [inline framework/jwt.go]: I think we should compute some portion of this.

Comment (hyfather) [inline framework/jwt.go]: Removed saasID

Comment (hyfather) [inline framework/jwt.go]: Yep, parsing the actual jwt will be done later on, this is only a dummy.

**Comment by bk10287** [comment]
OK, sounds good to me

**Pull request merged**